swaymsg/swaymsg.1.scd

```diff
@@ -51,7 +51,9 @@ _swaymsg_ [options...] [message]
 	  If you have quoted multi-word strings in your command, enclose the entire
 	  command in single-quotes. For example, use
 	  _swaymsg 'output "Foobar Display" enable'_ instead of
-	  _swaymsg output "Foobar Display" enable_.
+	  _swaymsg output "Foobar Display" enable_. Furthermore, note that comma
+      separated options also count as multi-word strings, because commas can be
```

Comment:
The indentation should use mixed tabs and spaces here: one tab followed by two spaces.

Reply:
Sorry, I trusted my editor's visual output, should have checked it more carefully. Thanks!